qbx_core.sql

```diff
@@ -11,12 +11,13 @@ CREATE TABLE IF NOT EXISTS `players` (
   `position` text NOT NULL,
   `metadata` text NOT NULL,
   `inventory` longtext DEFAULT NULL,
+  `phone_number` VARCHAR(20) DEFAULT NULL,
```

Comment:
Do we want to add the phone_number table in core?

Reply:
Oh, I didn't even see this. This doesn't seem related to the PR description

Reply:
My bad was using the recipe SQL as a reference, its also out of scope for this PR
https://github.com/Qbox-project/txAdminRecipe/blob/main/qbox.sql#L133 thanks for catching that